OpenRA.Mods.Common/Projectiles/Missile.cs

```diff
@@ -573,7 +573,7 @@ int IncreaseAltitude(int predClfDist, int diffClfMslHgt, int relTarHorDist, int
 				{
 					// Aim for the target
 					var vDist = new WVec(-relTarHgt, -relTarHorDist, 0);
-					desiredVFacing = (sbyte)vDist.HorizontalLengthSquared != 0 ? vDist.Yaw.Facing : vFacing;
+					desiredVFacing = (sbyte)(vDist.HasNonZeroHorizontalLength ? vDist.Yaw.Facing : vFacing);
```

Comment:
`vDist.Z` is guaranteed to be `0` here, so you can simply check for `vDist != WVec.Zero` instead.